k8s.io/canary.sh

```diff
@@ -0,0 +1,18 @@
+#!/bin/sh
+
+set -o errexit
+set -o nounset
+set -o pipefail
+
+alias kc="kubectl --context=utilicluster --namespace=k8s-io-canary"
+
+kc apply \
+    -f configmap-nginx.yaml \
+    -f configmap-www-get.yaml \
+    -f configmap-www-golang.yaml \
+    -f deployment.yaml \
+    -f service-canary.yaml
+
+sleep 5
```

Comment:
probably, but I didn't want to deal with it :)  will update

On Tue, Aug 23, 2016 at 12:00 PM, Ryan Hitchman <notifications@github.com>
wrote:

> In k8s.io/canary.sh
> <https://github.com/kubernetes/k8s.io/pull/4#discussion_r75929983>:
>
> > +#!/bin/sh
> > +
> > +set -o errexit
> > +set -o nounset
> > +set -o pipefail
> > +
> > +alias kc="kubectl --context=utilicluster --namespace=k8s-io-canary"
> > +
> > +kc apply \
> > +    -f configmap-nginx.yaml \
> > +    -f configmap-www-get.yaml \
> > +    -f configmap-www-golang.yaml \
> > +    -f deployment.yaml \
> > +    -f service-canary.yaml
> > +
> > +sleep 5
>
> Is there a better way to wait for the deployment to complete? 5 seconds
> may be too fast.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/kubernetes/k8s.io/pull/4/files/7c8000e9c6131f670e3457732d46204fdaae20f4#r75929983>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AFVgVNmx9EZy_Vl70p22MM3YXFEZAyCiks5qi0NegaJpZM4JqiuI>
> .
>